src/app/destiny2/d2-definitions.ts

```diff
@@ -142,22 +145,12 @@ export interface D2ManifestDefinitions extends ManifestDefinitions {
   DamageType: DefinitionTable<DestinyDamageTypeDefinition>;
   Collectible: DefinitionTable<DestinyCollectibleDefinition>;
   EventCard: DefinitionTable<DestinyEventCardDefinition>;
+  LoadoutConstants: DefinitionTable<DestinyLoadoutConstantsDefinition>;
   LoadoutName: DefinitionTable<DestinyLoadoutNameDefinition>;
   LoadoutColor: DefinitionTable<DestinyLoadoutColorDefinition>;
   LoadoutIcon: DefinitionTable<DestinyLoadoutIconDefinition>;
-
-  InventoryBucket: { [hash: number]: DestinyInventoryBucketDefinition };
-  Class: { [hash: number]: DestinyClassDefinition };
-  Gender: { [hash: number]: DestinyGenderDefinition };
-  Race: { [hash: number]: DestinyRaceDefinition };
-  Faction: { [hash: number]: DestinyFactionDefinition };
-  ItemTierType: { [hash: number]: DestinyItemTierTypeDefinition };
-  ActivityMode: { [hash: number]: DestinyActivityModeDefinition };
-  LoadoutConstants: { [hash: number]: DestinyLoadoutConstantsDefinition };
 }
 
-export const allTables = [...eagerTables, ...lazyTables];
```

Comment:
If you leave the `allTables` list in the same order, we can keep using the same cache. If they change, even in order, we'll force a redownload.